Library/Formula/synergy.rb

```diff
@@ -0,0 +1,75 @@
+class Synergy < Formula
+  homepage 'http://synergy-project.org'
+  url 'https://github.com/synergy/synergy/archive/1.6.3.tar.gz'
+  sha1 'c4c26ee983f0684a5cbe45520adc0b50a71a3524'
+
+  depends_on "cmake" => :build
+  depends_on "qt5" => :build
+
+  def install
+    ENV.deparallelize  # if your formula fails when building in parallel
+
+    system './hm.sh', 'conf', '-g1', '--mac-sdk', '10.10', '--mac-identity', 'test'
+    system './hm.sh', 'build'
+
+    bin.install 'bin/synergyc'
+    bin.install 'bin/synergyd'
+    bin.install 'bin/synergys'
+    bin.install 'bin/syntool'
+
+    prefix.install 'bin/Synergy.app'
+  end
+
+  patch :DATA
+
+  test do
+    system "#{bin}/synergys", '--version'
```

Comment:
well… we might make a quite complicated test by launching the server, then the client, then testing they connect by parsing both outputs :-s But that's quite a complicated thing to do…

I could expand the tests to check that all binaries have been correctly compiled, with --version on all.